ansible_base/rbac/validators.py

```diff
@@ -100,6 +100,11 @@ def validate_permissions_for_model(permissions, content_type: Optional[Model], m
         role_model = content_type.model_class()
     permissions_by_model = permissions_allowed_for_role(role_model)
 
+    if not managed:
+        if not permission_registry.object_roles_enabled(role_model):
+            print_model = role_model._meta.verbose_name if role_model else 'global roles'
+            raise ValidationError({'content_type': f'Creating roles for the {print_model} model is disabled'})
```

Comment:
Is this the right termonology in the exception? `the global roles model is disabled`.
Also, we should likely translate these strings if they can get raised to the browser.

Reply:
Yeah, the flow of this isn't good. The method `object_roles_enabled` unconditionally returns `True` if passed `None`, and the `None` value signifies a global role. So, it would be more clear to have this block be `if (not managed) and role_model:`, so we're only considering this condition when it's an object-based (as opposed to system) role definition.